.github/workflows/release-upload-to-wporg.yml

```diff
@@ -0,0 +1,227 @@
+name: 'Release: Upload release to WordPress.org'
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release to upload (can be a draft release).'
```

Comment:
Nit: would it make sense to say here "Release **version** to upload" or "Release **tag** to upload"?